lib/features/dashboard/presentation/widgets/dashboard_information.dart

```diff
@@ -0,0 +1,70 @@
+
+import 'package:design_system/design_system.dart';
+import 'package:flutter/material.dart';
+
+
+class DashboardCard extends StatelessWidget {
+  const DashboardCard({
+    super.key,
+    required this.message,
+    this.isLoading = false,
```

Comment:
this card is the same one as in the home. change the name to a more generic option and use the same one in both of them.

This change. Find the reference in home an migrate to the shared folder. delete DashboardCard. use the shared reference